src/dataDisplay/popover/__test__/popover.test.js

```diff
@@ -0,0 +1,225 @@
+/**
+ * To run this test from the church-management/client folder, execute the following command:
+ * npx jest ./src/dataDisplay/popover/__test__/popover.test.js
+ */
+import React from 'react';
+import mountWithTheme, {
+    waitForComponentToPaint,
```

Comment:
`waitForComponentToPaint()` is one of my favorite unit test utilities as of late.  It is _very_ useful!  👍